experiment/reporter.py

```diff
@@ -42,11 +42,12 @@ def output_report(web_bucket, in_progress=False):
         generate_report.generate_report([experiment_name],
                                         str(reports_dir),
                                         in_progress=in_progress)
-        gsutil.rsync(str(reports_dir),
-                     web_bucket,
-                     gsutil_options=[
-                         '-h', 'Cache-Control:public,max-age=0,no-transform'
-                     ])
+        bucket_utils.rsync(str(reports_dir),
+                           web_bucket,
+                           gsutil_options=[
```

Comment:
These options will probably be safe to ignore, but we may need to figure out a better solution in case we want to use options that aren't safe to ignore at some point.